### ISSUE TITLE
test message

### ISSUE THREAD

**Issue by spaette** [JuliaPlots/Plots.jl#4661]
This message could be rewritten to refer to 2 diacritical marks.

```
$ ed -s Plots.jl/test/test_pgfplotsx.jl <<<'428,433p'
    @testset "Latexify - LaTeXStrings" begin
        @test Plots.pgfx_sanitize_string("A string, with 2 punctuation chars.") ==
              "A string, with 2 punctuation chars."
        @test Plots.pgfx_sanitize_string("Interpolação polinomial") ==
              raw"Interpola$\textnormal{\c{c}}$$\textnormal{\~{a}}$o polinomial"
        @test Plots.pgfx_sanitize_string("∫∞ ∂x") == raw"$\int$$\infty$ $\partial$x"
$ 
```

**Comment by t-bltg** [comment]
I don't understand what you mean, could you rephrase / precise what your thoughts are ?

**Comment by spaette** [comment]
AFAIK, these are diacritical marks.

accent aigu
accent grave
accent circonflexe
cédille
tilde
tréma

test/test_pgfplotsx.jl

```diff
        @testset "Latexify - LaTeXStrings" begin
-         @test Plots.pgfx_sanitize_string("A string, with 2 punctuation chars.") ==
-               "A string, with 2 punctuation chars."
+         @test Plots.pgfx_sanitize_string("A string, with 2 diacritical marks.") ==
+               "A string, with 2 diacritical marks."
          @test Plots.pgfx_sanitize_string("Interpolação polinomial") ==
                raw"Interpola$\textnormal{\c{c}}$$\textnormal{\~{a}}$o polinomial"
          @test Plots.pgfx_sanitize_string("∫∞ ∂x") == raw"$\int$$\infty$ $\partial$x"
```

**Comment by t-bltg** [comment]
Hum, in the string `"A string, with 2 punctuation chars."` I see 2 punctuation chars: `,` and `.`, but no diacritics there ?

**Comment by spaette** [comment]
my gaffe

a side effect of dumping words from a repo and checking for typos

should have examined the code more closely before opening the ticket
